scripts/dev_haas.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+# Change the port and the username to match your enviorment
+Description=HaaS Dev Server listening on port 5000, for user sahil
+After=network.target
+Wants=
+
+[Service]
+#Change the username your username. same for group
+User=sahil 
+Group=sahil
```

Comment:
Maybe make the default User/Group "user"?

We could also make them "ubuntu", which matches the cloud image